packages/core/src/transformFeedback/TransformFeedbackSystem.ts

```diff
@@ -0,0 +1,160 @@
+import type { ISystem, IRenderingContext, Renderer, Shader } from '@pixi/core';
+import type { DRAW_MODES } from '@pixi/constants';
+import type { TransformFeedback } from './TransformFeedback';
+
+export class TransformFeedbackSystem implements ISystem
```

Comment:
Maybe would be good to add some doc/description here similar as for example in: https://github.com/pixijs/pixijs/blob/dev/packages/core/src/shader/UniformGroup.ts#L8-L50  ?

Reply:
I wrote it down but actually there's not much to say since all it does is just wrapping the TransformFeedback system of WebGL's 😂

Reply:
The "how" (how would you use this, examples, etc) and "why" (what use-cases does this solve) are often more important than "what" (what the API is)

Reply:
> The "how" (how would you use this, examples, etc) and "why" (what use-cases does this solve) are often more important than "what" (what the API is)

Updated!